packages/mgt-element/README.md

```diff
@@ -1 +1,93 @@
-# mgt-element
\ No newline at end of file
+# Microsoft Graph Toolkit Base package
+
+[![npm](https://img.shields.io/npm/v/@microsoft/mgt-element?style=for-the-badge)](https://www.npmjs.com/package/@microsoft/mgt-element)
+
+The [Microsoft Graph Toolkit (mgt)](https://aka.ms/mgt) library is a collection of authentication providers and UI components powered by Microsoft Graph. 
+
+The `@microsoft/mgt-element` package contains all base classes that enable the providers and components to work together. Use this package to set the global provider, or to create your own providers and/or components that work with Microsoft Graph.
+
+[See docs for full documentation](https://aka.ms/mgt-docs)
+
+## Set and use the global provider
+
+The `@microsoft/mgt-element` package exposes the `Providers` namespace that enables global usage of the authentication providers across your entire app. 
+
+This example illustrates how to instantiate a new provider (MsalProvider in this case) and use it across your app:
+
+1. Install the packages 
+
+    ```bash
+    npm install @microsoft/mgt-element @microsoft/mgt-msal-provider
+    ```
+
+1. Create the provider
+
+    ```ts
+    import {Providers} from '@microsoft/mgt-element';
+    import {MsalProvider} from '@microsoft/mgt-msal-provider';
+
+    // initialize the auth provider globally
+    Providers.globalProvider = new MsalProvider({clientId: 'clientId'});
+    ```
+
+1. Use the provider to sign in and call the graph:
+
+    ```ts
+    import {Providers, ProviderState} from '@microsoft/mgt-element';
+
+    const handleLoginClicked = async () => {
+      await Providers.globalProvider.login();
+
+      if (Providers.globalProvider.state === ProviderState.SignedIn) {
+        let me = await Provider.globalProvider.graph.client.api('/me').get();
+      }
+    }
+    ```
+
+You can learn more about how to use the providers in the [documentation](https://docs.microsoft.com/graph/toolkit/providers).
+
+The providers work well with the `@microsoft/mgt-components` package and all components use the provider automatically when they need to call the graph.
+
+## Create your own provider
+
+In scenarios where you want to use the Providers namespace and/or add mgt components to an application with pre-existing authentication code, you can create a custom provider that hooks into your authentication mechanism. `@microsoft/mgt` provides two ways to create new providers:
```

Comment:
```suggestion
In scenarios where you want to use the Providers namespace and/or add Microsoft Graph Toolkit components to an application with pre-existing authentication code, you can create a custom provider that hooks into your authentication mechanism. `@microsoft/mgt` provides two ways to create new providers:
```

Reply:
Should this be "`@microsoft/mgt-element` provides two ways to create new providers:"?